tests/test_wb_interpolate.py

```diff
@@ -40,16 +40,57 @@ def test_decoding():
     assert coeffs == int_msg
 
 
+def test_decoding_all_zeros():
+    int_msg = [0, 0, 0, 0, 0, 0, 0, 0]
+    k = len(int_msg)  # length of message
+    n = 22  # size of encoded message
+    p = 53  # prime
+    t = k - 1  # degree of polynomial
+
+    enc, dec, _ = make_encoder_decoder(n, k, p)
+    encoded = enc(int_msg)
+    # print("plain message is: %r" % (integerMessage,))
+    # print("encoded message is: %r" % (encoded,))  # cleaner output
+
+    # Check decoding with no errors
+    with pytest.raises(IndexError):
```

Comment:
Nice! Add a link to the bug here so we know which bug it is in case someone fixes this later.